pkg/security/probe/probe_bpf.go

```diff
@@ -222,7 +222,10 @@ func (p *Probe) SendStats(statsdClient *statsd.Client) error {
 		}
 	}
 
-	if err := statsdClient.Gauge(MetricPrefix+".process_resolver.cache_size", float64(len(p.resolvers.ProcessResolver.entryCache)), []string{}, 1.0); err != nil {
+	p.resolvers.ProcessResolver.RLock()
+	count := float64(len(p.resolvers.ProcessResolver.entryCache))
```

Comment:
I would prefer to add an accessor in the `ProcessResolver` struct, what about `GetCount` ?

Reply:
Done :+1: